src/components/LoanCards/KivaClassicBasicLoanCard.vue

```diff
@@ -150,7 +150,7 @@
 			v-if="allSharesReserved"
 			class="
 				tw-rounded
-				tw-bg-brand-100
+				tw-bg-primary-inverse
```

Comment:
Looks like the designs were changed to use grey so that's cool!

Reply:
huh, something's off. I would expect `tw-bg-primary-inverse` to look like a black background. Your screenshot looks like `tw-bg-secondary`
![image](https://user-images.githubusercontent.com/187937/134989001-9c58d34d-199e-4458-994c-30e853b1c398.png)

Reply:
Oddly, when I click on the [link you sent me earlier](https://608b4cf87f686c00213841b1-pkurhylpcj.chromatic.com/?path=/docs/base-styling-primitives--primitives) It shows a different mapping of these colors. 

🤔 
<img width="1057" alt="Screen Shot 2021-09-27 at 4 27 48 PM" src="https://user-images.githubusercontent.com/1521381/134993946-b40cf517-9883-495a-9d69-52a348513b7e.png">

Reply:
But looking at the headings, this page is broken, at least for me, so I'll update the class to be tw-bg-secondary.

Reply:
oh, I think you had dark mode turned on in that storybook (it's not obvious in "docs" view)
![image](https://user-images.githubusercontent.com/187937/134999787-cd9d3cdb-a61e-4ef9-b6b9-6cbade16855d.png)

Reply:
Thanks Ryan